test/presubmit-tests.sh

```diff
@@ -25,7 +25,10 @@ export NODE_DISTRO=linux-x64
 
 export KNATIVE_SERVING_VERSION=${KNATIVE_SERVING_VERSION:-latest}
 export KNATIVE_EVENTING_VERSION=${KNATIVE_EVENTING_VERSION:-latest}
-source $(dirname $0)/../vendor/knative.dev/hack/presubmit-tests.sh
+source "$(dirname "$0")/../vendor/knative.dev/hack/presubmit-tests.sh"
+
+FUNC_REPO_BRANCH_REF="$(git branch --show-current)"
```

Comment:
would that work? I mean.. the branch is on source repo.. maybe you should set the source repo as well.. 
Later on tests it shall resolve (for example) to `https://raw.githubusercontent.com/knative/func/test-builder-img/pkg/pipelines/resources/tekton/task/func-buildpacks/0.2/func-buildpacks.yaml` then it would fail

Reply:
I think you are right.

Reply:
or maybe set to commit hash (if reachable) so it would resolve to https://raw.githubusercontent.com/knative/func/b2f9dc538870f552ca2bc7393a88e3660c54a4c3/pkg/pipelines/resources/tekton/task/func-buildpacks/0.2/func-buildpacks.yaml

Reply:
@upodroid any idea how could I get repository in form of `org-or-user/repository` in the prow job?

Reply:
So hash works with `raw.githubusercontent.com` even when repository is not correct? Interesting.

Reply:
I'll use the `PULL_PULL_SHA` envvar.